documentation/docs/03-routing.md

```diff
@@ -315,6 +287,66 @@ export default config;
 
 > Using native `<form>` behaviour ensures your app continues to work when JavaScript fails or is disabled.
 
+### Standalone endpoints
+
+Most commonly, endpoints exist to provide data to the page with which they're paired. They can, however, exist separately from pages. Standalone endpoints have more flexibility — they return [`Response`](https://developer.mozilla.org/en-US/docs/Web/API/Response) objects. This means you have more possibilities for your response `body` type besides JSON, for example [`Uint8Array`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Uint8Array) or a [`ReadableStream`](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream).
+
+Standalone endpoints reside in a `+server.js` (or `.ts`) file, the folder describes the URL where the endpoint is available - just like for pages:
+
+| filename                              | endpoint   |
+| ------------------------------------- | ---------- |
+| src/routes/data/index.json/+server.js | /data.json |
+| src/routes/data/+server.js            | /data      |
+
+Because the foldername describes the endpoint URL, you can't have a `+server.js` file next to a `+page.svelte` file — when hitting the URL, SvelteKit wouldn't know if you want to render the page or return data from the endpoint.
+
+#### GET, POST, PUT, PATCH, DELETE
+
+Endpoints can handle any HTTP method — not just `GET` — by exporting the corresponding function:
+
+```js
+// @noErrors
```

Comment:
this is weird in example.

Reply:
This instructs wiki two slash, it's not visible in the output. You'll find some odd stuff like that in other places as well

Reply:
oh, ok, didn't know.